Tests/designspaceLib/data/test.designspace

```diff
@@ -38,6 +38,12 @@
                 <dimension name="width" xvalue="20" />
             </location>
         </source>
+        <source familyname="MasterFamilyName" filename="masters/masterTest2.ufo" layer="supports" name="master.ufo2" stylename="Supports">
```

Comment:
the "stylename" attibute in the case of support layers seems a bit redundant, even more so than for normal sources. I wonder if it could be omitted in these cases.